### PR TITLE
Inconsistency in Styling and Title Visibility for Top Visited and Recently Visited Cards

### PR DESCRIPTION
**Pull request by zeshanziya**
## Hey, I just made a Pull Request!

Please see Issue #23096 

#### Changes:
Modified the VisitList component to ensure proper rendering of the card title. Integrated the title from createCardExtension in plugin.ts to maintain consistency with other cards on the home page.

#### :heavy_check_mark: Checklist

Old UI
old

New UI:
new

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by fjudith** [comment]
Is it possible to review this PR please ! This is a blocker for ones who to translates their app.
As and example if you customize the titles in the `HomePage.tsx` titles gets duplicated in the UI.

image

**Review by fjudith — APPROVED** [review]

**Comment by zeshanziya** [comment]
Hi @callebstrom, have you had a chance to review the PR? Please let me know if you need any additional information from my side.

**Review by callebstrom — APPROVED** [review]
LGTM :+1:

**Review by tikabom — APPROVED** [review]

File: plugins/home/src/components/VisitList/VisitList.test.tsx [hunk 1]
```diff
@@ -19,16 +19,9 @@ import { VisitList } from './VisitList';
...
-    const { getByText } = await renderInTestApp(
-      <VisitList title="My title" detailType="time-ago" />,
-    );
-    expect(getByText('My title')).toBeInTheDocument();
-  });
-
   it('renders skeleton when loading is true', async () => {
-    const { container } = await renderInTestApp(
-      <VisitList title="My title" detailType="time-ago" loading />,
+    const { container } = await render(
```
Comment (awanlin) [inline plugins/home/src/components/VisitList/VisitList.test.tsx:23]: Is there a reason you aren't using `renderInTestApp` anymore? It's generally the recommended method. That goes for all the cases this was changed in this file.

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this @zeshanziya, this is just a high level review, the build is failing due to these changes so getting that green would be a good starting point. I think if you fix the test and run them locally to make sure they pass will be the first thing to do. Then that should get `yarn tsc` to pass, but also try that locally.

I'm pretty sure the API Report will change so please make sure to update that as well, details on how to do that are here: https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#api-reports

File: plugins/home/src/components/VisitList/VisitList.test.tsx (hunk 1 above)
Comment (zeshanziya) [reply plugins/home/src/components/VisitList/VisitList.test.tsx:23]: Thanks @awanlin, I will take a look and fix it. I didn't change much in the test cases. When I had pushed the changes, all test cases passed. Some days later, it started showing merge conflicts, indicating some other code had been merged.

I will fix the test cases as soon as possible and ensure they pass successfully.

Comment (zeshanziya) [reply plugins/home/src/components/VisitList/VisitList.test.tsx]: Hi @awanlin , Changes are done and test cases are passing now. Please review.

**Review by awanlin — APPROVED** [review]
Awesome, many thanks @zeshanziya 🚀

**Pull request merged**